sql/parser/sql.y

```diff
@@ -2807,6 +2808,14 @@ func_expr_common_subexpr:
 | GREATEST '(' expr_list ')' { unimplemented() }
 | LEAST '(' expr_list ')' { unimplemented() }
 
+func_or_variable:
```

Comment:
The variable form of `current_timestamp` is present in the SQL standard, and some databases forbid the use of parentheses with it (http://stackoverflow.com/questions/10084967/is-oracles-current-timestamp-function-really-a-function, although this information is suspect since it says that postgres disallows parens)